packages/devtools/lib/src/vm_service_wrapper.dart

```diff
@@ -445,6 +512,23 @@ class VmServiceWrapper implements VmService {
     activeFutures.clear();
   }
 
+  Future<bool> isProtocolVersionLessThan({
+    @required int major,
+    @required int minor,
+  }) async {
+    _protocolVersion ??= await getVersion();
+    return protocolVersionLessThan(major: major, minor: minor);
+  }
+
+  bool protocolVersionLessThan({
```

Comment:
Can this be hidden _protocolVersionLessThan and everyone calls isProtocolVersionLessThan?

Reply:
I had this originally, but then realized we need to call it from the constructor of ClassHeapDetailStats, which can't be async.